src/development/stack.yml

```diff
@@ -344,6 +350,162 @@ services:
       - postgres_user
     volumes:
       - ../../../reccoom/:/srv/app/
+
+  zammad-elasticsearch:
```

Comment:
Can elasticsearch be used by Zammad only or can we have it as `elasticsearch` only, similar to `postgres`?

Reading https://github.com/zammad/zammad-docker-compose?tab=readme-ov-file#running-without-elasticsearch it seems like elasticsearch could be excluded from the stack for "small teams". It appears some maintenance may be required for upgrades which I'm not sure how complicated they are. Please check that a bit further.